examples/unconditional_image_generation/train_unconditional.py

```diff
@@ -617,51 +627,53 @@ def transform_images(examples):
                 logs["ema_decay"] = ema_model.cur_decay_value
             progress_bar.set_postfix(**logs)
             accelerator.log(logs, step=global_step)
+        
+        loss_dict["loss_epoch_"+str(epoch)]= loss.detach().item()
```

Comment:
`loss` is already recorded before (in line 615). Why don't we reuse that instead?

Reply:
Yes, I'm using the loss calculated earlier in the code, and I'm creating a dictionary to save the loss for each epoch.